scripts/entrypoint.sh

```
@@ -93,7 +102,7 @@ if ! wait_for_ipam; then
 fi
 
 log_in_json info "Copying config file ... "
-
+validate_env_var
```

Lets move this func to even before starting IPamD so that we can fail fast ?